plugins/ua_pubsub_ethernet.c

```diff
@@ -1028,7 +1029,7 @@ sendWithTxTime(UA_PubSubChannel *channel, UA_ExtensionObject *transportSettings,
 static UA_StatusCode
 UA_PubSubChannelEthernet_send(UA_PubSubChannel *channel,
                               UA_ExtensionObject *transportSettings,
-                              const UA_ByteString *buf) {
+                              UA_ByteString *buf) {
```

Comment:
constness needs to be removed from all the `UA_ByteString` that will pass through the eventloop for sending, because the eventloop is expecting it.